### PR TITLE
Bugfix: Resolve issues around playlist

### PR DESCRIPTION
**Pull request by wutschel**
## Description

This PR applies follwing changes. 

- First change fixes an issue described in a [forum post](https://forum.kodi.tv/showthread.php?tid=359717&pid=3049725#pid3049725), which results in double `1x01` (or whatever season/episode is played) shown in the App's playlist when the playback was started via the Kodi UI. 
Screenshot: https://abload.de/img/a5137fe2-9c4b-4309-ap0k3p.jpeg

- Second change fixes missing or wrong thumbnails when calling the info screen via longpress on the playlist item.
Screenshots:
https://abload.de/img/simulatorscreenshot-i6iku0.png (TVShow before)
https://abload.de/img/simulatorscreenshot-i5lkam.png (TVShow after)
https://abload.de/img/simulatorscreenshot-it5ksx.png (Movie before)
https://abload.de/img/simulatorscreenshot-iqjkie.png (Movie after)
 
- Last changes are some coding style changes and the renaming of three main variables to match its content, which is not only Video based.

## Summary for release notes

Bugfix: Do not show double "1x01" in playlist for TVShow episodes
Bugfix: Show thumbnails for Movies/TVShows when calling info via playlist's longpress menu

### CODE REVIEW

File: XBMC Remote/NowPlaying.m [hunk 1]
```diff
@@ -2123,10 +2119,12 @@ - (UITableViewCell*)tableView:(UITableView*)tableView cellForRowAtIndexPath:(NSI
     else if ([item[@"type"] isEqualToString:@"movie"]) {
         [subLabel setText:[NSString stringWithFormat:@"%@", item[@"genre"]]];
     }
-    if (playerID == 0)
```
Comment (kambala-decapitator) [inline XBMC Remote/NowPlaying.m:2126]: in this case `switch` is better

File: XBMC Remote/NowPlaying.m [hunk 2]
```diff
@@ -2189,8 +2188,9 @@ - (void)tableView:(UITableView*)tableView didSelectRowAtIndexPath:(NSIndexPath*)
 }
 
 - (BOOL)tableView:(UITableView*)tableView canEditRowAtIndexPath:(NSIndexPath*)indexPath {
-    if (storeSelection && storeSelection.row == indexPath.row)
+    if (storeSelection && storeSelection.row == indexPath.row) {
```
Comment (kambala-decapitator) [inline XBMC Remote/NowPlaying.m:2191]: you could simply return the inverted condition (or even "open" the brackets):

    return !(storeSelection && storeSelection.row == indexPath.row);

but not insisting on this.

**Review by kambala-decapitator — CHANGES_REQUESTED** [review]

File: XBMC Remote/NowPlaying.m (hunk 1 above)
Comment (wutschel) [reply XBMC Remote/NowPlaying.m:2126]: Done in fixup commit.

File: XBMC Remote/NowPlaying.m (hunk 2 above)
Comment (wutschel) [reply XBMC Remote/NowPlaying.m]: Done in fixup commit.

**Comment by wutschel** [comment]
Let me know if this is ok now. I will squash the changes then.

**Review by kambala-decapitator — APPROVED** [review]

**Comment by wutschel** [comment]
Here we go.

**Pull request merged**